Assets/Tests/InputSystem/Editor/CoreEditorTests.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections;
+using System.IO;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.LowLevel;
+using UnityEngine.TestTools;
+
+internal class CoreEditorTests
+{
+    [UnityTest]
+    [Category("Actions")]
+    public IEnumerator Actions_InteractiveRebinding_NewBindingsAreRemovedFromAssetOnExitingPlayMode()
```

Comment:
This will have to cover more than just newly added bindings. Binding overrides and all other action/binding setup changes should also be nuked. If, for example, at runtime, code adds a new control scheme and bindings for it, all the data should also disappear.

Reply:
Overrides are already ignored since the override strings aren't serialized, but I added code to deal with new control schemes

Reply:
>Overrides are already ignored since the override strings

I think the tests should explicitly check for that, though. The current implementation does remove them as a side-effect but if the implementation changes and no longer does that, would be great to have a test immediately go red.

Would prefer a single test that checks all mutations in one go.

* Added/deleted/changed bindings (both overrides and non-overrides)
* Added/deleted/changed actions
* Added/deleted/changed action maps
* Added/deleted/changed control schemes

IMO the easiest way would actually be to just clone the original asset, then apply a couple mutations, and then exit out of play mode and compare the JSON of the clone to the JSON of the original asset and expect them to match.

Reply:
Ok, added tests for all asset changes I could find. Let me know if I missed any. I also moved the tests back to play mode because they probably make more sense there now.